man/org.freedesktop.hostname1.xml

```diff
@@ -287,6 +287,17 @@ node /org/freedesktop/hostname1 {
     purpose of those properties is to allow remote clients to access this information over D-Bus. Local
     clients can access the information directly.</para>
 
+    <para><varname>MachineID</varname> expose the 128bit machine ID, see
+    <citerefentry><refentrytitle>machine-id</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+    details.</para>
+
+    <para><varname>BootID</varname> expose the 128bit boot ID, as per
+    <filename>/proc/sys/kernel/random/boot_id</filename>.</para>
+
+    <para><varname>VSockContextIdentifier</varname> exposes the system's local <constant>AF_VSOCK</constant>
```

Comment:
I know we usually don't abbreviate, but since we already have MachineID= and BootID= here, wouldn't it be consistent to name this VsockCID=?